vnext/Desktop/BeastWebSocketResource.cpp

```diff
@@ -51,92 +47,89 @@ BaseWebSocketResource<Protocol, SocketLayer, Stream, Resolver>::~BaseWebSocketRe
   }
 }
 
-template <typename Protocol, typename SocketLayer, typename Stream, typename Resolver>
-void BaseWebSocketResource<Protocol, SocketLayer, Stream, Resolver>::Handshake(
-    const IWebSocketResource::Options &options) {
-  m_stream->async_handshake_ex(
-      m_url.host,
-      m_url.Target(),
-      // Header handler
-      [options = std::move(options)](websocket::request_type &req) {
-        // Collect headers
-        for (const auto &header : options) {
-          req.insert(Microsoft::Common::Unicode::Utf16ToUtf8(header.first), header.second);
-        }
-      },
-      // Handshake handler
-      [this](boostecr ec) {
-        if (ec) {
-          if (m_errorHandler)
-            m_errorHandler({ec.message(), ErrorType::Handshake});
-        } else {
-          m_handshakePerformed = true;
-          m_readyState = ReadyState::Open;
-
-          if (m_connectHandler)
-            m_connectHandler();
-
-          // Start read cycle.
-          PerformRead();
-
-          // Perform writes, if enqueued.
-          if (!m_writeRequests.empty())
-            PerformWrite();
-
-          // Perform pings, if enqueued.
-          if (m_pingRequests > 0)
-            PerformPing();
-
-          // Perform close, if requested.
-          if (m_closeRequested && !m_closeInProgress)
-            PerformClose();
-        }
-      }); // async_handshake_ex
-}
-
-template <typename Protocol, typename SocketLayer, typename Stream, typename Resolver>
-void BaseWebSocketResource<Protocol, SocketLayer, Stream, Resolver>::PerformRead() {
+template <typename SocketLayer, typename Stream>
+void BaseWebSocketResource<SocketLayer, Stream>::Handshake() {
+  ////TODO: Enable?
```

Comment:
Just checking if this TODO is intentional to check in or if this one you wanted to address before merging.

Reply:
It's intentional. It doesn't seem to affect client behavior, but it was introduced in the new APIs.